### DISCUSSION TITLE
RemoteApp support

### DISCUSSION THREAD

**Discussion by mashhoorgulati** [baSSiLL/SharpAvi#60] [Q&A]
Hi,

I'm using SharpAVI in a console application. So the idea is once the console app is launched it starts the recording and stops once app is closed. 

This works perfectly on local desktop. As well as on the Remote Dektop(mstsc) also. But if the application is used through RemoteApp, where only the application is launched on successful RDP connection and the whore desktop.

When used in RemoteApp, the RDP connects but doesn't log on and time outs after 2 minutes(default for remoteapp start failure).

Any idea with this.?

--
Thanks.

**Comment by baSSiLL** [comment]
Hi
First of all, what does your app record?
Anyway, I've never used RemoteApp before so barely can give you a specific advice. Start with identifying which activity in the program causes an issue.
